api/alembic/versions/0d46262707af_alter_morecast_forecast_columns_to_float.py

```diff
@@ -0,0 +1,34 @@
+"""Alter morecast_forecast columns to float
+
+Revision ID: 0d46262707af
+Revises: 69cbd7ca2477
+Create Date: 2023-03-14 15:55:31.325373
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0d46262707af'
+down_revision = '69cbd7ca2477'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column('morecast_forecast', 'temp')
+    op.drop_column('morecast_forecast', 'precip')
+    op.drop_column('morecast_forecast', 'wind_speed')
+    op.add_column('morecast_forecast', sa.Column('temp', sa.Float(), nullable=False))
```

Comment:
My general thoughts on this table (I doubt there's any action required in this PR for them):
- forecasters *should* be forecasting a wind direction, but they don't always. Should check with PO/stakeholders to see whether Morecast 2 should be enforcing this or not. Not allowing a forecaster to submit a forecast just because wind dir is missing might be problematic
- this table will probably need to be updated in the future to add a (nullable?) column for grass cure.

Reply:
Most definitely!